xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoPPFFmpeg.cpp

```diff
@@ -54,7 +54,7 @@ bool CDVDVideoPPFFmpeg::CheckInit(int iWidth, int iHeight)
       Dispose();
     }
 
-    m_pContext = pp_get_context(iWidth, iHeight, PPCPUFlags() | PP_FORMAT_420);
+    m_pContext = pp_get_context(iWidth, iHeight, PP_CPU_CAPS_AUTO | PP_FORMAT_420);
```

Comment:
For the PP_FORMAT_420 - not so sure, as when we are doing SW decoding chances are hight that we also cope with 444, 422 or others.

Reply:
Seems FFmpeg post-processing only is enabled for 420 video:

https://github.com/xbmc/xbmc/blob/f7cb67356da26a890a89847aae6c9360f2db26c9/xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoPPFFmpeg.cpp#L89-L96

Hence the PP_FORMAT_420 flag is always used.